.github/workflows/translators-sync.yml

```
@@ -0,0 +1,46 @@
+name: Sync Translators Branch
+
+permissions:
+  actions: write
+  checks: write
+  contents: write
+  deployments: write
+  pull-requests: write
+  statuses: write
+  
+on:
+  push:
+    branches:
+        - 'master*'
+        
+jobs:
+    sync-translation:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout 🛎️
+              uses: actions/checkout@v2.3.1
+            - name: Setup Node
+              uses: actions/setup-node@v2.1.2
```

```suggestion
              uses: actions/setup-node@v4.0.0
```

https://github.com/actions/setup-node